lib/rules/block-closing-brace-newline-after/__tests__/index.js

```diff
@@ -207,6 +207,37 @@ testRule(rule, {
   ]
 });
 
+testRule(rule, {
+  ruleName,
+  config: ["always", { ignoreBeforeAtRules: "/else/" }],
+
+  accept: [
+    {
+      code: `@if ($var) {
+        color: pink;
+      } @else if ($var2) {
+        color: red;
+      } @else {
+        color: blue;
+      }
+      `
+    },
+    {
+      code: `@media (width < 28em) {
+        background: red;
+      } @else {
+        background: green;
+      }`
+    }
+  ],
+  reject: [
```

Comment:
@hudochenkov could we please update tests README with suggested reject test case style:
```
warnings: [..]
```
to avoid problem with mismatching amount of warnings

Reply:
@rambleraptor could you please rewrite rejected test cases as follow https://github.com/stylelint/stylelint/blob/0bd166f57814946b81515a9d96b9313078b27d11/lib/rules/selector-max-pseudo-class/__tests__/index.js#L189

Reply:
> to avoid problem with mismatching amount of warnings

@vankop I don't think we really needed. Our tests are quite verbose already. Having `warnings: []` with only one warning inside will make them more verbose. When [strict check](https://github.com/stylelint/stylelint/pull/4231) will be enabled, we won't miss any warnings.

Reply:
you are correct, I did some work toward it =)